lua/methods/unit.lua

```diff
@@ -16,43 +16,34 @@ GRPC.methods.getRadar = function(params)
   local active, object = unit:getRadar() 
   
   if object == nil then
-    env.info("[GRPC]" .. params.name .. " has no radar target")
     return GRPC.success({   
       active = active
     })
   end
-  
-  env.info("[GRPC]" .. params.name .. "'s Target is " .. object:getCallsign() .. ", ID: " .. object:getID() .. ", Category: " .. object:getCategory() )
    
   local category = object:getCategory()
-  local grpc_hash = {}
+  local grpc_table = {}

```

Comment:
I am wondering whether we should keep it to camelCase in the Lua file? No strong opinion though

Reply:
From what I have read snake_case and camelCase is personal preference. I used snake_case because ruby influence but will switch to camelCase because that is what the ED APIs use.